src/Symfony/Component/Routing/Tests/Fixtures/AnnotationFixtures/FooController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Symfony\Component\Routing\Tests\Fixtures\AnnotationFixtures;
+
+use Symfony\Component\Routing\Annotation\Route;
+
+class FooController
+{
+    /**
+     * @Route("/Blog")
+     */
+    public function simplePath()
+    {
+    }
+
+    /**
+     * @Route({"nl":"/hier","en":"/here"})
+     */
+    public function localized()
+    {
+    }
+
+    /**
+     * @Route("", requirements={"locale":"en"})
+     */
+    public function requirements()
+    {
+    }
+
+    /**
+     * @Route({"requirements":{"locale":"en"}, "path":""})
```

Comment:
Okay, right. This does work now that we've enabled `@NamedArgumentConstructor`, but I'd consider this to be an accident rather than intended behavior. I would not build a test case for that.

Reply:
related to https://github.com/symfony/symfony/pull/40767/files#r612721813. This fixture is to test deprecation but maybe I misunderstand the target of the deprecation.

Reply:
The deprecation is not supposed to be triggered by normal annotation usage. The target are manual constructor calls.

Reply:
Oh ok thank you. I pushed some changes :)

Reply:
Sorry, I used cs fixer with a wrong PHP version;it broke attributes. I'll fix it tomorrow.

Reply:
Updated :)